### DISCUSSION TITLE
Slick Window Arrangement is the best customization I've used

### DISCUSSION THREAD

**Discussion by Ayane9** [ramensoftware/windhawk-mods#716] [General]
I use a 3440x1440 monitor alongside an elder 1080p monitor. I used multiple windows all the time, browser for general browsing, you tube, a random app or two, settings, etc. 

This has taken a TON of the tedium of resizing and moving around windows. Especially when I want the window to line up exactly with the edges of the monitor/other windows.

I just wanted to share my thanks. Really great mod.

**Comment by m417z** [comment]
You're welcome, thanks for the great feedback! :)

**Comment by jmain33** [comment]
This post persuaded me to give this one a try. The name is a bit eccentric and from the preview & description I couldn't discern what sliding windows meant. But it didn't hurt to give it a try.

Snapping at the borders is really useful. I just wish there was a choice to let them snap from inside each other - when the windows are overlapping - in addition to the standard way.
